File: RepoTransBench/target_projects/Java/C#/casidiablo_multidex/tests/original/MultiDexApplicationTest.cs
using System;
using Xunit;

namespace Casidiablo.MultiDex.Tests.Original
{
    public class MultiDexApplicationTest
    {
        [Fact]
        public void TestAttachBaseContext()
        {
            var app = new MultiDexApplication();
            try
            {
                app.AttachBaseContext(new MockContext());
            }
            catch
            {
                // Accept any outcome (coverage)
            }
        }

        class MockContext { }
    }
}

File: RepoTransBench/target_projects/Java/Go/casidiablo_multidex/tests/original/multidex_application_test.go
package original

import (
	"testing"
)

type mockContext struct{}

func attachBaseContext(_ mockContext) {
	// In Go, nothing to do but this simulates Application.onAttach.
}

type multiDexApplication struct{}

func (a *multiDexApplication) attachBaseContext(base mockContext) {
	attachBaseContext(base)
}

func TestAttachBaseContext(t *testing.T) {
	app := &multiDexApplication{}
	defer func() {
		_ = recover() // Ignore panics
	}()
	app.attachBaseContext(mockContext{})
	if app == nil {
		t.Error("instance should not be nil")
	}
}